index.js

```diff
@@ -100,23 +129,31 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
+
+function triple(threeTimes) {
+  console.log(threeTimes * 3)
+}
 
 // console.log(myMap([1,2,3,4,5], triple)) 
 // // => [3,6,9,12,15]
 
 // console.log(myMap([2,4,6,8], triple))   
 // // => [6,12,18,24]
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 // ***** Callbacks - Question 2 *****
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 2", "color: red")
+console.log("%cCallbacks - Question 2", "color: red")
+
+function greet(nameOne, nameTwo) {
+  console.log(`"Hi, ${nameOne}!", "Hi, ${nameTwo}!"`)
+}
```

Comment:
- same issue here as the previous question with needing a return value.
- myMap is expecting to be able to call this function passing in one argument at a time. So this should be: 
```javascript
function greet(nameOne) {
  console.log(`Hi, ${nameOne}!`)
}
```